includes/Core/Mail/Hooks.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace WeDevs\WeMail\Core\Mail;
+
+use WeDevs\WeMail\Traits\Hooker;
+
+class Hooks {
+
+    use Hooker;
+
+    /**
+     * Hooks constructor.
+     */
+    public function __construct()
+    {
```

Comment:
The parenthesis should be on the same line.

Reply:
fixed